server/engine/src/main/antlr4/org/eclipse/lsp/cobol/implicitDialects/sql/Db2SqlExecParser.g4

```diff
@@ -516,16 +521,37 @@ organization_clause: ORGANIZE BY HASH UNIQUE column_loop partition_hash_space?;
 
 //CREATE TABLESPACE
 dbs_create_tablespace: TABLESPACE dbs_table_space_name dbs_create_tablespace_opts*;
-dbs_create_tablespace_opts : IN (DSNDB04  | dbs_database_name) | BUFFERPOOL dbs_bp_name | partition_by_growth_spec  | partition_by_range_spec |
-               SEGSIZE INTEGERLITERAL | DSSIZE dbs_dsize_parameter  | CCSID oneof_encoding | CLOSE yes_or_no | COMPRESS no_or_yes | DEFINE no_or_yes | free_block  |  gbpcache_block
-               | INSERT ALGORITHM dbs_create_algorithm_level | LOCKMAX (SYSTEM | INTEGERLITERAL) | locksize_block_tbl  | TRACKMOD yes_or_no | using_block;
+dbs_create_tablespace_opts : IN (DSNDB04  | dbs_database_name)
+               | partition_by_growth_spec
+               | partition_by_range_spec
+               | DEFINE no_or_yes
+               | locksize_block_tbl
+               | MAXROWS INTEGERLITERAL
+               | MEMBER CLUSTER
+               | DSSIZE dbs_dsize_parameter
+               | CCSID oneof_encoding
+               | SEGSIZE INTEGERLITERAL
+               | dbs_create_alter_tablespace_opts_common
+               | using_block;
+dbs_create_alter_tablespace_opts_common: BUFFERPOOL dbs_bp_name
```

Comment:
could the CCSID, MAXROWS, MAXPARTITIONS, SEGSIZE MEMBER CLUSTER clauses be included here?

Reply:
Not included because of below points
CCSID -> create takes input as CCSID (ASCII | EBCDIC | UNICODE) , while alter takes CCSID integer
MAXROWS -> **updated**
MAXPARTITIONS -> create can take a sub argument of numparts
SEGSIZE -> alter when has a this argument, no other clauses can be specified. Removed from common rule for this behavior
MEMBER CLUSTER -> create takes this argument, but alter need a subcommand of (YES|NO)